src/client/scripts/cardGrid.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+/*global Cards:false Rooms:false */
+
+Template.cardGrid.onRendered(function(){
+    Session.set("getUniqueID_CallCount",0);
+    Session.set("pairSet",false);
+});
+
+Template.cardGrid.helpers({
+
+    validMultiRow: function(){
+        var categories = Rooms.findOne(
+          {"roomCode": Session.get("roomNumber")}
+      ).categories;
+        var len = categories.length;
+
+        return len > 4;
+    },
+
+    getRow: function(){
+        var rowOfCategories = [];
+        var categories = Rooms.findOne(
+          {"roomCode": Session.get("roomNumber")}
+      ).categories;
+        var index = 0;
+        var numInnerArrays = Math.floor(4 / categories.length);
+
+        numInnerArrays += 4 % categories.length;
```

Comment:
you are using this magic number 4 everywhere.  I'm guessing it is to set the number of categories per row (or something similar) to 4.  Have a var at the top and declare it there.